test/visualize/test_visualize.py

```diff
@@ -256,19 +256,19 @@ def test_parameters_with_options():
 
     # test calls with specific options
     visualize.parameters(result_1,
-                         free_indices_only=False,
+                         parameter_indices='all',
                          reference=ref_point,
                          size=alt_fig_size,
                          colors=[1., .3, .3, 0.5])
 
     visualize.parameters([result_1, result_2],
-                         free_indices_only=False,
+                         parameter_indices='all',
                          reference=ref_point,
                          balance_alpha=False,
                          start_indices=(0, 1, 4))
 
     visualize.parameters([result_1, result_2],
-                         free_indices_only=True,
+                         parameter_indices='all',
```

Comment:
I think here it should be 'free_only'...
Would be good to us in the above example a list of indices, to also have the new functionality checked.

Reply:
Oh, you're right, thanks!